examples/nextjs/starter/package.json

```diff
@@ -4,7 +4,7 @@
 	"private": true,
 	"type": "module",
 	"scripts": {
-		"dev": "concurrently \"next dev\" \"npm run codegen:watch\"",
+		"dev": "concurrently \"NODE_OPTIONS='--inspect' next dev\" \"npm run codegen:watch\"",
```

Comment:
EILI5?

Reply:
> Is it needed for this PR?

Nope. I was having a hard time debugging in the terminal. I can remove it.